src/components/Event/EventView.js

```diff
@@ -116,10 +116,10 @@ const EventView = ({ event, children }) => {
           <div />
         )}
 
-        <div className={classes.registrationHeader}>
+        {!isRegistered && <div className={classes.registrationHeader}>
```

Comment:
can we move the conditional logic and line 119 - 122 to eventregister/index? does it make more sense to keep all that logic all in the same file? that way the isRegistered value doesn't need to be passed to EventView either